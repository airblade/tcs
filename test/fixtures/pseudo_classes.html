<div class="hover:opacity-75 opacity-50 hover:scale-150 scale-125">
</div>

<div class="focus:opacity-75 opacity-50 hover:scale-150 scale-125">
</div>
