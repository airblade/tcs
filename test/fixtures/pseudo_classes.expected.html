<div class="scale-125 opacity-50 hover:scale-150 hover:opacity-75">
</div>

<div class="scale-125 opacity-50 hover:scale-150 focus:opacity-75">
</div>
